lambda_functions/healthcheck/healthcheck.py

```diff
@@ -18,12 +13,17 @@ def check_web_url_health(url, expected_string, env):
     :type expected_string: String
     :param env: This is for us to distinguish the message between the dev, preprod and prod environments
     :type env: string
-    :raises logger.error: will log an error to the lambda log group
-    :raises logger.error: variable
+    :raises RuntimeError: will raise a runtime error if appropriate
+    :raises RuntimeError: error
+    :raises Exeption: will raise a runtime error if appropriate
```

Comment:
You have raises runtimeError and raises exception error "will raise a runtime error if appropriate" - do these comments make sense?

Reply:
done :)